packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -190,7 +190,7 @@ const UsersProfileFn = ({terms, slug, classes}: {
       </div>
     }
 
-    if (!user || !user._id || user.deleted) {
+    if (!user || !user._id || (user.deleted && !currentUser?.isAdmin)) {
       //eslint-disable-next-line no-console
       console.error(`// missing user (_id/slug: ${slug})`);
       return <Error404/>
```

Comment:
Are you keeping this change on purpose? Maybe there should be some indication on the profile page that the user is deleted, to avoid confusion?

Also I think if you keep this, we should port it over to the `EAUsersProfile` as well, just for consistency